setup.py

```diff
@@ -29,7 +29,7 @@
     "numproto==0.3",  # Apache License 2.0
     "structlog==19.2.0",  # Apache License 2.0
     # TODO: change xain-proto requirement to "xain-proto==0.2.0" once it is released
-    "xain-proto @ git+https://github.com/xainag/xain-proto.git@0e52b2fd1ceabbcccd443b05e2438a9ce0c65178#egg=xain_proto-0.2.0&subdirectory=python",  # Apache License 2.0
+    "xain-proto @ git+https://github.com/xainag/xain-proto.git@master#egg=xain_proto-0.1.0&subdirectory=python",  # Apache License 2.0
```

Comment:
IMO we should pin `xain-proto` to a specific revision to prevent breakage when it get updated.

Reply:
i agree with this in general. pointing to a branch `@ git+http...` is only a temporary solution anyway, would it be ok to leave as it is right now and fix this properly to `"xain-proto==0.2.0"` once it got released in the near future?

Reply:
done.